spec/predicates/vsa.md

```diff
@@ -1,11 +1,279 @@
 
-Version: 1.0
+Version: 1.1
 
 ## Purpose
 
-Records a SLSA verification decision about an artifact.
+Verification summary attestations communicate that an artifact has been verified
+against some policy and details about that verification. Such details may
+include, but are not limited to, what SLSA level the artifact and/or its
+dependencies were verified to meet.
 
-For more details, please see https://slsa.dev/spec/v1.0/verification_summary
+## Use Cases
+
+Software consumers make a decision about the validity of an
+artifact without needing to have access to all of the attestations about the
+artifact or all of its transitive dependencies.  They delegate
+complex policy decisions to some trusted party and then simply trust that
+party's decision regarding the artifact. That decision is represented by a VSA.
+
+VSAs also allow software producers to keep the details of their build pipeline
+confidential while still communicating that some verification has taken place.
+This might be necessary for legal reasons (keeping a software supplier
+confidential) or for security reasons (not revealing that an embargoed patch has
+been included).
+
+## Prerequisites
+
+Understanding of [in-toto specification] and the in-toto attestation framework.
+
+## Model
+
+A Verification Summary Attestation (VSA) is an attestation that some entity
+(`verifier`) verified one or more software artifacts (the `subject` of an
+in-toto attestation [Statement]) by evaluating the artifact and a `bundle`
+of attestations against some `policy`.  Users who trust the `verifier` may
+assume that the artifacts satisfied the indicated security polciy without
+themselves needing to evaluate the artifact or to have access to the
+attestations the `verifier` used to make its determination.
+
+The VSA also allows consumers to determine the verified levels of
+all of an artifact’s _transitive_ dependencies.  The verifier does this by
+either a) verifying the provenance of each non-source dependency listed in
+the [resolvedDependencies](/provenance/v1#resolvedDependencies) of the artifact
+being verified (recursively) or b) matching the non-source dependency
+listed in `resolvedDependencies` (`subject.digest` ==
+`resolvedDependencies.digest` and, ideally, `vsa.resourceUri` ==
+`resolvedDependencies.uri`) to a VSA _for that dependency_ and using
+`vsa.verifiedProperties` and `vsa.dependencyProperties`.  Policy verifiers
+wishing to establish minimum requirements on dependencies may use
+`vsa.dependencyProperties` to do so.
+
+## Schema
+
+```jsonc
+// Standard attestation fields:
+"_type": "https://in-toto.io/Statement/v1",
+"subject": [{
+  "name": <NAME>,
+  "digest": { <digest-in-request> }
+}],
+
+// Predicate
+"predicateType": "https://slsa.dev/verification_summary/v1",
+"predicate": {
+  "verifier": {
+    "id": "<URI>",
+    "version": {
+      "<COMPONENT>": "<VERSION>",
+      ...
+    }
+  },
+  "timeVerified": <TIMESTAMP>,
+  "resourceUri": <artifact-URI-in-request>,
```

Comment:
optional: Would this be a good time to consider if we should change this name?

It's a bit confusing that this _field name_ also happens to match [this _type name_](https://github.com/in-toto/attestation/blob/99d851228fe284c66b2cde353a6693c5eff69db1/spec/v1/field_types.md?plain=1#L12).

Would `verifiedUri` be better??

Reply:
That's what I originally thought when we were developing this internally, but @AdamZWu made some compelling arguments for separating it.  At least one of those arguments was that it seemed to violate layering.

I think we also recently ran across a case and thought "oh wow, it's a good thing we have this separate field and don't have it in the subject".  Maybe something to do with having VSAs that have _lots_ of subjects but really only one resourceUri applies.  If it were in the subject it needs to be repeated for each digest even if it's the same thing and that can result in very large attestations.   @AdamZWu should more more details.